### PR TITLE
QOL and structural improvements

### PR DESCRIPTION
**Pull request by TABASCOatw**
Changes made:
- Structural and textual shift to focus on WaaS (in progress)
- Landing page optimized (to be further optimized post-release)
- `@particle-network/auth` removed from main Web Auth SDK page.
- Quickstart guide for Web consolidated with previous page and structure; to be replicated across other quickstart pages.
- Web3 Services API documentation removed.
- WaaS Components page removed.
- Optimizations made to backlinks.

### CODE REVIEW

File: landing/introduction.mdx [hunk 1]
```diff
@@ -7,52 +7,43 @@ iconType: "light"
 
 ## What is Particle Network?
 
-Particle Network allows developers to easily integrate social logins and account abstraction in decentralized applications, enhancing user experience and streamlining development.
+Particle Network radically simplifies user experience through Wallet Abstraction and Chain Abstraction. Through its Wallet-as-a-Service SDKs, developers can enable 2-click onboarding into smart accounts through social logins.
```
Comment (TABASCOatw) [inline landing/introduction.mdx:10]: Optimizing with Carlos.

File: landing/introduction.mdx [hunk 2]
```diff
@@ -7,52 +7,43 @@ iconType: "light"
...
-
-- **[Particle Connect](../api-reference/connect):** Connect to Web 2.0 accounts and mainstream wallets, enhancing accessibility through a unified interface.
-- **[Particle Auth](../api-reference/auth):** Onboard users with one-click authentication using popular platforms like Google, X, and many more.
-- **[Embedded Wallet](../api-reference/wallet):** Integrate a Web3 wallet directly into your dApp, allowing users to manage their assets without leaving your application.
-- **[Account Abstraction](../api-reference/aa):** Modularize your dApp into ERC-4337 AA, leveraging Particle’s Bundler, Paymaster, and compatibility with Smart Account implementations.
-- **[BTC Connect](../api-reference/btc):** Control a Smart Account on an EVM-compatible L2 using your Bitcoin wallet.
+Particle Network current product suite features numerous SDKs made to simplify authentication and onboarding:
 
 <CardGroup cols={2}>
+  <Card title="Particle Auth" icon="shield" href="../api-reference/auth">
```
Comment (TABASCOatw) [inline landing/introduction.mdx:17]: List of links were redundant, only using cards looks better.

File: landing/introduction.mdx [hunk 3]
```diff
@@ -7,52 +7,43 @@ iconType: "light"
...
 
 ## Connect with Particle Network
 
-Stay up to date with Particle Network and reach out to us at any time for questions, issues, and feedback.
+Stay up to date with Particle Network. Reach out to us anytime with questions, issues, or feedback.
 
 <CardGroup cols={3}>
-  <Card title="Particle Network Twitter" icon="twitter" href="https://x.com/ParticleNtwrk">
-    Follow us on Twitter to stay updated with the latest news and announcements.
+  <Card title="@ParticleNtwrk" icon="x-twitter" href="https://x.com/ParticleNtwrk">
```
Comment (TABASCOatw) [inline landing/introduction.mdx:43]: Not too sure about these; saying "Main X" "Developer X" or "Main Twitter" "Developer Twitter" felt off. Perhaps there's a better way to do this

File: landing/what-is-particle-network.mdx [hunk 4]
```diff
@@ -8,7 +8,13 @@ iconType: "light"
 
 ***
 
-## Particle Network: The L1 Unifying All Chains Through Universal Accounts
+# Particle Network, Existing Products: Wallet-as-a-Service
+
+(section in progress)
```
Comment (TABASCOatw) [inline landing/what-is-particle-network.mdx:13]: Being written; done before docs release

File: resources/quickstart/customization/auth.mdx [hunk 5]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Customizing Particle Auth"
+title: "Customizing Login Modal"
```
Comment (TABASCOatw) [inline resources/quickstart/customization/auth.mdx:2]: Preparing for more general terminologies here with the new Particle Connect SDK

File: resources/quickstart/social-logins.mdx
Comment (TABASCOatw) [inline resources/quickstart/social-logins.mdx:1]: Merged with web-quickstart

File: resources/quickstart/waas/web-quickstart.mdx [hunk 6]
```diff
@@ -95,72 +89,51 @@ To retrieve these values for configuration within your application, follow these
...
-          visible: true,
-        },
-      }}
-    >
-      <App />
-    </AuthCoreContextProvider>
-  </React.StrictMode>
-)
-```
+import { InteractiveLayout } from '/snippets/InteractiveLayout.mdx';
```
Comment (TABASCOatw) [inline resources/quickstart/waas/web-quickstart.mdx:100]: This needs to be replicated for Mobile WaaS quickstart, Desktop AA quickstart, Mobile AA quickstart, and the customization pages (these can wait until new Particle Connect SDK is ready).

Not a huge priority but definitely the goal to have consistent interactive config snippets within these more granular quickstarts

File: resources/quickstart/waas/web-quickstart.mdx [hunk 7]
```diff
@@ -95,72 +89,51 @@ To retrieve these values for configuration within your application, follow these
...
 });
 ```
 
+<Tip>The complete Web SDK reference is available at [Web (JavaScript/TypeScript)](/api-reference/auth/desktop-sdks/web). Additionally, a demo repository for Next.js can be found [here](https://github.com/Particle-Network/particle-auth-ethers-demo/tree/main/particle-next-starter), alongside a demo for create-react-app [here](https://github.com/Particle-Network/particle-auth-ethers-demo/tree/main/particle-auth-cra).</Tip>
+
 ***
 
-<Tip>The complete Web SDK reference is available at [Web (JavaScript/TypeScript)](/api-reference/auth/desktop-sdks/web).</Tip>
\ No newline at end of file
+### Related Resources
```
Comment (TABASCOatw) [inline resources/quickstart/waas/web-quickstart.mdx:127]: Similar degrees of consistent cross-linking resources needs to be replicated across pages; should be as simple as possible for developers to find what they want (such as customization).

File: landing/introduction.mdx [hunk 8]
```diff
@@ -7,52 +7,43 @@ iconType: "light"
 
 ## What is Particle Network?
```
Comment (TABASCOatw) [inline landing/introduction.mdx:8]: This page is okay as-is, but after launch, need to make it sharper with some nice opening GIF or interactive embedded widget. Something to generate immediate understanding about the product

File: resources/quickstart/waas/web-quickstart.mdx (hunk 6 above)
Comment (soos3d) [reply resources/quickstart/waas/web-quickstart.mdx:100]: yep, this is definitely the goal.

File: landing/introduction.mdx (hunk 2 above)
Comment (soos3d) [reply landing/introduction.mdx:17]: yep, left both to choose from

**Pull request merged**